pkg/ddc/vineyard/utils_test.go

```diff
@@ -420,8 +420,8 @@ func TestParseMasterImage(t *testing.T) {
 			tag:             "",
 			imagePullPolicy: "",
 			engine:          &VineyardEngine{},
-			wantImage:       "bitnami/etcd",
-			wantTag:         "3.5.10",
+			wantImage:       "registry.aliyuncs.com/vineyard/vineyardd",
```

Comment:
Are you using the same docker image for etcd and vineyard?

Reply:
Yes, we found the image `bitnami/etcd` has many security issues thus we replaced it with the vineyard image as it has a built-in etcd binary.